contracts/IdentityFactory.sol

```diff
@@ -11,7 +12,7 @@ contract IdentityFactory {
     mapping(address => address) public senderToProxy;
 
     //cost ~2.4M gas
-    function CreateProxyWithControllerAndRecovery(address userKey, address[] delegates, uint longTimeLock, uint shortTimeLock) {
+    function createProxyWithControllerAndRecovery(address userKey, address[] delegates, uint longTimeLock, uint shortTimeLock) {
```

Comment:
Not sure about the mobile app, but this would need updating in the identity creation server. However this won't be until we make a new release to master, and hopefully the next release will deprecate this in favor of the identity manager.